src/applicationMicroservices/orderSercice/src/main/java/com/shop/ordersercice/dao/MyConnection.java

```diff
@@ -0,0 +1,19 @@
+package com.shop.ordersercice.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class MyConnection {
+    private static final String DB_URL = "jdbc:mysql://127.0.0.1:3306/shop";
+    private static final String DB_USER = "root";
+    private static final String DB_PASSWORD = "1111";
```

Comment:
Move this to application.property file and inject here via @Value

Reply:
I cant use @Value with static fields, but I can use setters) I rewrote this part of the code.